charts/kamaji/crds/kamaji.clastix.io_tenantcontrolplanes.yaml

```diff
@@ -6415,8 +6415,18 @@ spec:
                   description: |-
                     DataStore allows to specify a DataStore that should be used to store the Kubernetes data for the given Tenant Control Plane.
                     This parameter is optional and acts as an override over the default one which is used by the Kamaji Operator.
-                    Migration from a different DataStore to another one is not yet supported and the reconciliation will be blocked.
+                    Migration from a different DataStore to another one is supported, see: https://kamaji.clastix.io/guides/datastore-migration/
```

Comment:
This is incorrect; rather, this sentence should explain we do not offer migration between Datastore backed by different drivers.